rskj-core/src/main/java/org/ethereum/rpc/FilterManager.java

```diff
@@ -71,6 +72,11 @@ public int registerFilter(Filter filter) {
 
     public boolean removeFilter(int id) {
         synchronized (filterLock) {
+
+            Filter filter = installedFilters.get(id);
+            if (filter == null) {
+                throw new RskJsonRpcRequestException(-32000, "filter not found");

```

Comment:
what about adding a method in `RskJsonRpcRequestException` as it's done for other cases, eg. `public static RskJsonRpcRequestException filterNotFound()`?